Test failure: disaster_recovery_3

https://github.com/AntelopeIO/spring/actions/runs/15332471249/job/43143156436

```
Traceback (most recent call last):
  File "/__w/spring/spring/build/tests/disaster_recovery_3.py", line 102, in <module>
    assert currentLIB == n_LIB, f"Node {node.nodeId} advanced LIB {currentLIB} beyond N LIB {n_LIB}"
AssertionError: Node 2 advanced LIB 123 beyond N LIB 122
2025-05-29T20:22:36.390283 Shutting down wallet manager process 143
```

This is on a development branch with changes to `producer_plugin` start block. But best I can tell it looks like an issue with not giving enough time to pause.

--- a/tests/disaster_recovery_3.py
+++ b/tests/disaster_recovery_3.py
@@ -83,7 +83,7 @@ try:
     assert node0.waitForProducer("defproducera"), "Node 0 did not produce"
     for node in [node0, node1]:
         node.processUrllibRequest("producer", "pause", exitOnError=True)
-    time.sleep(0.5)
+    node0.waitForLibNotToAdvance()
 
     currentLIB = node0.getIrreversibleBlockNum()
     n_LIB = currentLIB + 1

